### PR TITLE
feat(sms): Enable Send SMS for 2.5% of the general population.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
The experiment is enabled for 5% of the population, 2.5% will be
in the control group, 2.5% in the treatment group.

softvision.ro is also enabled.

### Why the duplication of `isEmailInTreatment`?
I couldn't figure out an easy way to share code between both functions, so the logic is duplicated.

### Why doesn't this contain the "gate by country" code?
That's done [in the content server](https://github.com/mozilla/fxa-content-server/pull/4827/files#diff-0dc8124c36f55037d013efdc72d8eaa8R244). If this experiment pans out, we'll remove the experiment but will want to keep the auth-server gating logic.

### CODE REVIEW

File: send-sms-install-link/send-sms-install-link.js [hunk 1]
```diff
@@ -4,29 +4,48 @@ module.exports = {
...
-    return false;
+    // a random sampling of 5% of all sessions.
+    // 2.5% will be in the control group,
+    // 2.5% in the treatment group.
+    return this.bernoulliTrial(0.05, subject.uniqueUserId);
+
+    function isInExperiment(forceExperiment, email) {
+      return forceExperiment === 'sendSms' ||
+             // The regexps are duplicated in the groupingFunction because
+             // there is no way to share code between the two.
```
Comment (philbooth) [inline send-sms-install-link/send-sms-install-link.js]: I must confess, I've read this comment three times now and I feel like I'm going mad. Why can't we just move `isEmailInTreatment` so it's in scope for both functions?

Comment (vladikoff) [reply send-sms-install-link/send-sms-install-link.js]: > so it's in scope for both functions?

AFAIR you cannot access that scope from these functions

Comment (philbooth) [reply send-sms-install-link/send-sms-install-link.js]: 😯 ...? Aren't they just closures like anything else? Are they parsed independently or something? I'd quite like to undestand as this repo seems completely opaque and magical to me at the moment...

Comment (shane-tomlinson) [reply send-sms-install-link/send-sms-install-link.js]: >  Aren't they just closures like anything else? Are they parsed independently or something? I'd quite like to undestand as this repo seems completely opaque and magical to me at the moment...

They aren't. There's a lot of foo that goes on to create the browser bundle. The bundler only pulls things it expects, any function outside of module.exports is ignored.

Comment (philbooth) [reply send-sms-install-link/send-sms-install-link.js]: Ahhhh, got it. Thanks for explaining!

**Comment by philbooth** [comment]
LGTM!

File: send-sms-install-link/send-sms-install-link.js [hunk 2]
```diff
@@ -4,29 +4,48 @@ module.exports = {
...
       return true;
     }
 
-    return false;
+    // a random sampling of 5% of all sessions.
+    // 2.5% will be in the control group,
+    // 2.5% in the treatment group.
+    return this.bernoulliTrial(0.05, subject.uniqueUserId);
+
+    function isInExperiment(forceExperiment, email) {
```
Comment (vladikoff) [inline send-sms-install-link/send-sms-install-link.js]: I try to not put anything after a `return`, but up to you! 🍦

**Comment by vladikoff** [comment]
One style nit @shane-tomlinson , r+ otw

**Pull request merged**